src/qt/rpcconsole.cpp

```diff
@@ -459,11 +460,22 @@ RPCConsole::RPCConsole(interfaces::Node& node, const PlatformStyle *_platformSty
 
     ui->splitter->restoreState(settings.value("PeersTabSplitterSizes").toByteArray());
 
-    QChar nonbreaking_hyphen(8209);
+    const QChar nonbreaking_hyphen(8209);
+    const std::vector<QString> CONNECTION_TYPE_DOC{
```

Comment:
Named it `CONNECTION_TYPE_DOC` to show up in git grep with the constant of the same name in `rpc/net.cpp`